### ISSUE TITLE
Remove references to _addContinue and _removeContinue from Defer.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8534]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8534|
|Type|enhancement|
|Created|2016-06-29 11:29:28Z|
|Branch|https://github.com/twisted/twisted/tree/8534-rodrigc-defer|

In the doc string for Defer.py, there are references to _addContinue() and _removeContinue() functions,
which do not exist.  These references should be removed.

<details><summary>Searchable metadata</summary>

```
trac-id__8534 8534
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__None None
branch__8534_rodrigc_defer 8534-rodrigc-defer
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1467199768942456 1467199768942456
changetime__1467313195721205 1467313195721205
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/266

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban commented|
|-|-|

Looks good. Please merge. Thanks!

I have triggered the buildbots

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)|@adiroiban set owner to @rodrigc|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub set status to `closed`|
|-|-|

In changeset f43b16c097309d9747a0c1e708bd4ed05a6977e8
```
#!CommitTicketReference repository="" revision="f43b16c097309d9747a0c1e708bd4ed05a6977e8"
Merge 8534-rodrigc-defer-2: Remove docstring references to _addContinue() and _removeContinue()

Author: rodrigc
Reviewer: adiroiban
Fixes: #8534
```
